app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -58,6 +59,9 @@ public void onFailure(final Call<AlbumSearchResponse> call, final Throwable t) {
 
         } catch (IOException e) {
             e.printStackTrace();
+        } catch (NullPointerException e) {
```

Comment:
im not sure what would happen for it to return null, but android studio was warning me that something could of thrown a NullPointer and thats the only reason i could think it could return  that